addons/l10n_in_edi/models/account_move.py

```diff
@@ -235,9 +264,11 @@ def _l10n_in_edi_send_invoice(self):
                         body=_(
                             "Somehow this invoice has been submited to government before."
                             "%(br)sNormally, this should not happen too often"
-                            "%(br)sJust verify value of invoice by upload json to government website %(link)s.",
+                            "%(br)sJust verify value of invoice by upload json to government website %(link)s."
+                            "%(br)s%(note)s",
                             br=Markup("<br/>"),
-                            link=link
+                            link=link,
+                            note=not jwt and "We can't verify this invoice because JWT python library is not installed" or ""
```

Comment:
This is not user friendly... Most users won't get what this message is about and even less be able to fix anything about it.